pkg/quarterdeck/api/v1/client.go

```diff
@@ -348,6 +348,21 @@ func (s *APIv1) UserUpdate(ctx context.Context, in *User) (out *User, err error)
 	return out, nil
 }
 
+func (s *APIv1) UserRoleUpdate(ctx context.Context, id string, in *UpdateRoleRequest) (out *User, err error) {
+	endpoint := fmt.Sprintf("/v1/users/%s", id)
```

Comment:
Also RE: my above comment; having the role as a ULID means that we can guarantee the endpoint is created correctly. You would just need to do something like:

```
endpoint := fmt.Sprintf("/v1/users/%s", in.ID)
```